centered_eight.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<p>It looks like you have an old version of ArviZ or xarray installed, the default in latest ArviZ is an html representation for InferenceData objects. Can you try updating them? i.e. running <code>pip install -U arviz xarray</code>  (or with conda if you are using conda)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>